app/views/pages/personal_dashboard.html.erb

```diff
@@ -1 +1,59 @@
-<h1>Personal dashboard</h1>
+<div class="container">
+  <h1>Personal dashboard</h1>
+      <div class="personal-details">
+        <h2>My Account</h2>
+        <img src="" alt="avatar" class="rounded-circle"><br>
+        <br>
+           <%= current_user.first_name %><br>
+           <%= current_user.last_name %><br>
+           <%= current_user.address %><br>
+           <%= current_user.email %><br>
+           <br>
+          <button type="button" class="btn"></button>
+          <%= link_to 'edit', edit_user_registration_path %>
+
+          <button type="button" class="btn"></button>
+          <%= link_to 'delete', cancel_user_registration_path %>
+      </div>
+
+      <div class="user-plan">
+        <h2>User Plans</h2>
+        <ul>
+          <!-- list users userplans -->
+          <% @plans.each do |user_plan| %>
```

Comment:
Does this have to say @user_plan.each do ?

Reply:
for some reason user_plan would not render in the page
may be due to the fact that I am listing plans...and using <li> here